byterun/win32.c

```diff
@@ -925,13 +925,14 @@ void caml_init_os_params(void)
   GetSystemInfo(&si);
   caml_sys_pagesize = si.dwPageSize;
   QueryPerformanceFrequency(&frequency);
+  /* Convert the frequency to the duration of 1 tick in ns */
+  frequency.QuadPart = 1000000000L / frequency.QuadPart;
```

Comment:
@stedolan 128bit arithmetic will be bad - is converting the tick frequency to ns acceptable?